lib/msal-angular/docs/v2-v3-upgrade-guide.md

```diff
@@ -0,0 +1,51 @@
+# Upgrading from MSAL Angular v2 to v3
+
+MSAL Angular v3 brings our Angular wrapper up-to-date with the latest version of MSAL common, and with out-of-the-box support for Angular 15 and rxjs 7.
+
+This guide will demonstrate changes needed to migrate an existing application from `@azure/msal-angular` v2 to v3. 
+
+If you are migrating from `@azure/msal-angular` v1, please refer to the [v1-v2 migration guide](./v2-docs/v1-v2-upgrade-guide.md) first to migrate to MSAL v2.
+
+Please also see the [MSAL Browser Migration Doc](../../msal-browser/docs/v2-migration.md) for browser support and other key changes.
+
+## Breaking changes in `@azure/msal-angular@3`
+
+### `MsalRedirectComponent` is now mandatory
+
+MSAL v3.x now requires initializing the application object. Initialization has been built into the `MsalRedirectComponent`, bootstrapping the component is now **MANDATORY**.
+
+See the [guide to redirects](./v2-docs/redirects.md) on how to add `MsalRedirectComponent` to your application.
+
+### The `allowNativeBroker` flag
+
+The `allowNativeBroker` flag is now turned on by default in the configurations. To turn it off, configure your app as follows:
```

Comment:
Please remove the section on how to disable it.  If you feel really strongly about leaving it in, please update the comment to say ''// Disables native brokering support" vs "// Disables WAM Broker"

Reply:
I'd suggest the messaging around turning it off be focused on unsupported scenarios e.g.

```suggestion
The `allowNativeBroker` flag is now turned on by default in the configurations. If you're using a B2C authority you can turn it off as follows:
```